src/reactComponents/AddTabDialog.tsx

```diff
@@ -138,34 +112,13 @@ export default function AddTabDialog(props: AddTabDialogProps) {
 
     const newModule = storageProject.findModuleByClassName(props.project, newClassName);
     if (newModule) {
-      const module: Module = {
-        path: newModule.modulePath,
+      const newTab: TabItem = {
+        key: newModule.modulePath,
         title: newModule.className,
         type: tabType,
       };
-      setSelectedItems([...selectedItems, module]);
-      triggerProjectUpdate();
-    }
-
-
-    setNewItemName('');
-  };
-
-  /** Moves an item from available to selected list. */
-  const handleSelectItem = (item: Module): void => {
-    const existingItem = selectedItems.find((i) => i.path === item.path);
-    if (existingItem) {
-      return;
+      props.onOk(newTab);
```

Comment:
Can you clear the newItemName so that the next time this dialog is viewed, it doesn't have the previously used name?